cmd/opm/alpha/bundle/validate.go

```diff
@@ -36,7 +42,25 @@ func validateFunc(cmd *cobra.Command, args []string) error {
 	logger := log.WithFields(log.Fields{"container-tool": imageBuilderArgs})
 	log.SetLevel(log.DebugLevel)
 
-	imageValidator := bundle.NewImageValidator(imageBuilderArgs, logger)
+	var (
+		registry image.Registry
+		err      error
+	)
+
+	tool := containertools.NewContainerTool(imageBuilderArgs, containertools.NoneTool)
```

Comment:
Right now the help text for this flag lists docker and podman and is a "build" tool. Can we update the help text to at least mention the `none` option? I think if we did that we should also probably attempt to rename the parameter, since we aren't building anything in bundle validate.

Reply:
👍